card/spec/models/card/trash_spec.rb

```diff
@@ -239,18 +239,18 @@
   end
 
   it "test_recreate_plus_card_name_variant" do
-    Card.create( :name => "rta+rtb" ).delete
-    Card["rta"].update_attributes :name=> "rta!"
-    c = Card.create! :name=>"rta!+rtb"
+    Card.create( name: "rta+rtb" ).delete
+    Card["rta"].update_attributes name: "rta!"
+    c = Card.create! name: "rta!+rtb"
     assert Card["rta!+rtb"]
     assert !Card["rta!+rtb"].trash
     assert Card.find_by_key('rtb*trash').nil?
   end
 
   it "test_multiple_trash_collision" do
-    Card.create( :name => "alpha" ).delete
+    Card.create( name: "alpha" ).delete
     3.times do
-      b = Card.create( :name => "beta" )
+      b = Card.create( name: "beta" )
```

Comment:
Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.